<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewDownloadDialog</class>
 <widget class="QDialog" name="NewDownloadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>230</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Download Directory:</string>
       </property>
       <property name="buddy">
        <cstring>editDir</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="editDir"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="buttonBrowseDir">
       <property name="text">
        <string>&amp;Browse...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&amp;Category:</string>
       </property>
       <property name="buddy">
        <cstring>comboCategory</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="comboCategory">
       <item>
        <property name="text">
         <string>Video</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Audio</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Compressed</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Document</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Application</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Other</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&amp;URLs:</string>
       </property>
       <property name="buddy">
        <cstring>editUrls</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QPlainTextEdit" name="editUrls">
       <property name="placeholderText">
        <string>Input one URL per line.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonMagnetToTorrent">
       <property name="text">
        <string>&amp;Magnet to torrent</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editUrls</tabstop>
  <tabstop>editDir</tabstop>
  <tabstop>buttonBrowseDir</tabstop>
  <tabstop>comboCategory</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewDownloadDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewDownloadDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
